src/main/controlMod.F90

```diff
@@ -205,7 +205,7 @@ subroutine control_init(dtime)
          crop_fsat_equals_zero, for_testing_run_ncdiopio_tests, &
          for_testing_use_second_grain_pool, for_testing_use_repr_structure_pool, &
          for_testing_no_crop_seed_replenishment, &
-         z0param_method, use_z0m_snowmelt
+         z0param_method, use_z0m_snowmelt, dust_emis_method
```

Comment:
Maybe remove this as it isn't complete at least needing a broadcast. But, will be removed when this comes from CMEPS.

Reply:
It is required for now as it's set in the namelist. I'll add a comment to note that it will be removed and moved to inside of CMEPS, so the normal broadcast and error checking aren't needed here, only allowing it to be in the namelist.

Reply:
Comment added.